<?xml version="1.0"?>
<launch>



    <include file="$(find setup)/launch/p3at_spawn.launch">
      <arg name="robotname" value="robot3"/>
      <arg name="initialy" value="-7"/>
      <arg name="initialx" value="-41"/>
    </include>


    <include file="$(find setup)/launch/p3at_spawn.launch">
      <arg name="robotname" value="robot0"/>
      <arg name="initialy" value="7"/>
      <arg name="initialx" value="-41"/>
    </include>


</launch>
